libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -823,6 +836,32 @@ void CompilerUtils::cleanHigherOrderBits(IntegerType const& _typeOnStack)
 		m_context << ((u256(1) << _typeOnStack.numBits()) - 1) << Instruction::AND;
 }
 
+void CompilerUtils::cleanHigherOrderBits(FixedPointType const& _typeOnStack)
+{
+	if (_typeOnStack.numBits() == 256)
+		return;
+	else if (_typeOnStack.isSigned())
```

Comment:
okay, so I tried to convert them to integer types from fixed point types and that gave me a seg fault....so....maybe there is a case for at the very least editting the cleaning function?